pkg/conversions/conversions_test.go

```diff
@@ -0,0 +1,37 @@
+package conversions
+
+import (
+	"testing"
+
+	"github.com/alecthomas/units"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestConversions(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Conversions tests Suite")
+}
+
+var _ = Describe("bytes to string conversion", func() {
```

Comment:
It would be great to have a few tests for uneven amounts (i.e. 1229 B, etc.).

Reply:
Sure. I'll add a case :smile: